base/allocator/allocator_shim_override_libc_symbols.h

```diff
@@ -18,15 +18,15 @@
 
 extern "C" {
 
-SHIM_ALWAYS_EXPORT void* SbMemoryAllocate(size_t size) __THROW {
+SHIM_ALWAYS_EXPORT void* malloc(size_t size) __THROW {
```

Comment:
Did you ensure that changes made to 3p libraries match the upstream? We want to make sure to change it back to the original, not just do a mass rename

Reply:
Nope, this is out of scope for the initial API changes in SB16. 

The revert of the customizations is a separate effort and would follow up the initial PRs with the mass renames.